shared/src/main/scala/coop/rchain/shared/PathOps.scala

```diff
@@ -38,4 +42,49 @@ object PathOps {
         FileVisitResult.CONTINUE
       }
     }
+
+  implicit class PathDelete(path: Path) {
+    def delete[F[_]: Sync: Log](): F[Unit] = {
+      import java.io.File
+      import java.util.Comparator
+
+      import scala.collection.JavaConverters._
+
+      import cats.implicits._
+
+      def deleteFile(file: File)(implicit logSource: LogSource): F[Unit] =
+        for {
+          deleted <- Sync[F].delay(file.delete)
+          _ <- if (deleted) Log[F].info(s"Deleted file ${file.getAbsolutePath}")
+              else Log[F].warn(s"Can't delete file ${file.getAbsolutePath}")
+        } yield ()
+
+      def getFiles: F[List[Path]] =
```

Comment:
I don't like the behavior change being made in one commit with the move - it's hard to see what's actually being changed. Please keep that in mind for future PRs.